### PR TITLE
Report emoji events to the admin channel

### LINKED ISSUE #7: Add emoji reporting to the admin channel
## Description
Reinstate emoji reporting to the admin channel.

With this feature, the following should happen:
1. An emoji is added, updated, or removed
1. The application receives an event with the information on the emoji
1. The application sends a message through automod to the admin channel to give details about the event

## Blocked By ⛔️ 
- [x] Write ticket to get testing bot up and running (#9)
- [x] Get testing bot up and running

## Acceptance Criteria
- [ ] Application can receive events for when an emoji is added, removed, or when a new alias is created
- [ ] Bot posts a message about the event in the admin channel

## Extra
[Here](https://github.com/jpmcb/OSU-Auto-Mod/) is a link to the old automod repo for how this used to be implemented. [Here](https://github.com/slackapi/python-slackclient/tree/master/tutorial) is a link to getting started with a slack bot. [Here](https://api.slack.com/events/emoji_changed) is the event api we will most likely use.

### PR DESCRIPTION
**Pull request by hschallh**
Resolves #7 

## Description
Subscribes the application to the 'emoji_changed' event and sends off a message to the admins when the callback is hit.

## AC
- [ ] We can receive events for when an emoji is added, removed, or when a new alias is created
- [ ] Bot posts a message about the event in the admin channel

## Testing 
1. TODO: get test bot set up first

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -1,15 +1,43 @@
...
+import os
+import ssl as ssl_lib
+import certifi
+import slack
+from emoji_message import EmojiMessage
 
-app = Flask(__name__)  # pylint: disable=invalid-name
 
+@slack.RTMClient.run_on(event='emoji_changed')
+def onboarding_message(**payload):
```
Comment (sarahmaas) [inline app.py]: Naming confused me for a second :sweat_smile: onboarding made me think of the old feature for when someone joined the workspace.

Comment (hschallh) [reply app.py]: DEFINITELY NOT COPY AND PASTED FROM TUTORIAL

File: emoji_message.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    """Constructs emoji report message."""
+
+    def __init__(self, channel, icon_emoji, event_type):
+        """Initialize EmojiMessage.
+
+        Created with the channel to send the message to, the event type that triggered the callback and the emoji to
+        report on.
+
+        Arguments:
+            channel {string}    -- The channel to send the emoji message to
```
Comment (sarahmaas) [inline emoji_message.py]: Stick with str, not string for these I think. Later we can add actual typing. :smiling_imp:

Comment (hschallh) [reply emoji_message.py]: auto gen /shrug

**Review by sarahmaas — APPROVED** [review]

**Pull request merged**